adding upgrade for docker-compose version to support buildkit

codersofthedark opened the pull request.

codersofthedark committed:
adding upgrade for docker-compose version to support buildkit

Commit:
adding sudo

himadrisingh commented on scripts/travis-init.sh:
```diff
@@ -40,6 +40,11 @@ sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubun
+
 # enable buildkit
 export DOCKER_BUILDKIT=1
 
```
```suggestion
export COMPOSE_DOCKER_CLI_BUILD=1
```

Commit:
making script more readable

- made docker-compose version configurable
- added reference link for compatibility
- added script usage command as commenmt

Commit:
made docker-compose version configurable

codersofthedark committed:
adding COMPOSE_DOCKER_CLI_BUILD

Co-Authored-By: Himadri Singh <himadri.singh@gmail.com>

codersofthedark committed:
Update travis-init.sh
